jenkins/common/sonic-mgmt-framework-build-pr/Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+pipeline {
+    agent { node { label 'sonic-slave' } }
+
+    stages {
```

Comment:
Also add the option numToKeepStr ?

Reply:
None of pr jobs have numToKeepStr.  Is it some default value of 5 picked up for these jobs ?

lvnvde1934#pwd
/sonic-build-tools/jenkins/common

**lvnvde1934#grep -r numToKeep *pr
lvnvde1934#**

lvnvde1934#grep -r numToKeep *
dep-build/Jenkinsfile:        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10'))
linux-kernel-build/Jenkinsfile:        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10'))
python-asyncsnmp-build/Jenkinsfile:        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10'))
sonic-mgmt-framework-build/Jenkinsfile:        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10'))
sonic-swss-common-build/Jenkinsfile:        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10'))
sonic-swss-common-build-ubuntu/Jenkinsfile:        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10'))
sonic-telemetry-build/Jenkinsfile:        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10'))